libs/langchain/tests/integration_tests/vectorstores/test_redis.py

```diff
@@ -40,30 +72,55 @@ def test_redis_new_vector(texts: List[str]) -> None:
     """Test adding a new document"""
     docsearch = Redis.from_texts(texts, FakeEmbeddings(), redis_url=TEST_REDIS_URL)
     docsearch.add_texts(["foo"])
-    output = docsearch.similarity_search("foo", k=2)
+    output = docsearch.similarity_search("foo", k=2, return_metadata=False)
     assert output == TEST_RESULT
     assert drop(docsearch.index_name)
 
 
 def test_redis_from_existing(texts: List[str]) -> None:
     """Test adding a new document"""
-    Redis.from_texts(
+    docsearch = Redis.from_texts(
```

Comment:
I'd add tests that create or connect to an existing index that doesn't include the default fields names